Sources/ProjectDescription/CopyFilesAction.swift

```diff
@@ -0,0 +1,33 @@
+import Foundation
+
+public struct CopyFilesAction: Equatable, Codable {
+    public var name: String
+    public var destination: Destination
+    public var subpath: String
+    public var files: [FileElement]
+
+    public enum Destination: UInt, Codable, Equatable {
+        case absolutePath = 0
```

Comment:
where are these numbers coming from? Are we sure that this list covers all use-cases?

Reply:
Also - do we need these raw values in `ProjectDescription`?

Reply:
These values map directly to [PBXCopyFilesBuildPhase.swift#L5](https://github.com/tuist/XcodeProj/blob/main/Sources/XcodeProj/Objects/BuildPhase/PBXCopyFilesBuildPhase.swift#L5) on `func generateCopyFilesBuildPhase(path:` and it covers all the use-cases. You are right we don't need the raw values in `ProjectDescription`. I'm going to update that.